### ISSUE TITLE
Tests fail due to version number mismatch

### ISSUE THREAD

**Issue by sjamaan** [ursetto/atom-egg#1]
The first test [fails](https://salmonella-freebsd-x86-64.call-cc.org/master/clang/freebsd/x86-64/2019/09/14/salmonella-report/test/atom.html).

The fix is very simple: replace
```
  <generator uri="http://3e8.org/chickadee/atom" version="0.1.1">atom egg for Chicken</generator>
```
with
```
  <generator uri="http://3e8.org/chickadee/atom" version="0.1.3">atom egg for Chicken</generator>
```

**Comment by sjamaan** [comment]
Maybe it's a good idea to somehow inject the current version in the string instead of hardcoding it in the tests, as that would break the next time a new version is pushed (if you forget to run the tests first).

**Comment by ursetto** [comment]
Yes, it should definitely be injected. It looks like when this was updated from 0.1.1 to 0.1.2 in 2011, `+atom-egg-version+` was not updated with it, so the tests didn't fail. But, the test has been failing since 0.1.3. I will either have bumpversion inject the correct version, or generate the string in code.

**Comment by ursetto** [comment]
So, tests were not implemented until 0.1.3. But I ran the tests *before* bumping the version to 0.1.3, and they succeeded with the old version. Now it will be updated on bump.
